docs/Deployment/Hack0.md

```diff
@@ -0,0 +1,65 @@
+# Hack0
+
+Hack0 is an alternative to [Raspberry Pi](/Deployment/RaspberryPi.md) deployments.
+It greatly simplifies the installation steps by providing the hardware and the operating system to run your BTCPay Server with docker deployment.
+
+This project is maintained by [DG Lab](https://www.dglab.com/en/), if you need help, come on our [support chat](https://chat.btcpayserver.org/btcpayserver/channels/hack0).
+
+Hack0 aims at two different kind of users: `Distributors` and `End users`.
+
+* `End users` are people ultimately running the Hack0 for their own purposes.
+* `Distributors` are people who buy the different part of the hardware, assemble it together, install Hack0, and distribute a plug-and-play box for `End users`.
+
+If you buy the different piece of hardware for your Hack0, assemble, install hack0, then use it yourself, you are both a `distributor` and a `end users` as far as this documentation is concerned.
+
+You can watch an introduction here:
+
+[![Introduction to Hack0](https://img.youtube.com/vi/m3i2EUTEukM/mqdefault.jpg)](https://www.youtube.com/watch?v=m3i2EUTEukM "Introduction to Hack0")
+
+## Hardware specification (for distributors)
+
+Here is the advised parts for running a Hack0:
+
+* RockPro64 4GB ([Link](https://store.pine64.org/?product=rockpro64-4gb-single-board-computer)) `79.99$`
+* USB adapter for EMMC Module ([Link](https://pine64.com/product/usb-adapter-for-emmc-module/)) `4.99$`
+* EMMC 16GB ([Link](https://pine64.com/product/16gb-emmc-module/)) `15.95$`
+* Fan For ROCKPro64 20mm Mid Profile Heatsink ([Link](https://store.pine64.org/?product=fan-for-rockpro64-20mm-mid-profile-heatsink)) `2.99$`
+* ROCKPro64 20mm Mid Profile Heatsink ([Link](https://store.pine64.org/?product=rockpro64-20mm-mid-profile-heatsink)) `3.29$`
+* SSD 500GB PCIe NVMe, Crucial P1 ([Link](https://www.crucial.com/usa/en/ct500p1ssd8)) `66.99$`
+* Shift M.2 to PCIe adapter ([Link](https://shop.shiftcrypto.ch/en/products/compact-m2-nvme-ssd-to-pcie-adapter-15/)) `25$`
```

Comment:
Correction because ShiftCrypto doesn't sell these anymore: 

SilverStone M.2 SSD Port to PCIe X4 Conversion Interface Card SST-ECM25 `([Link](https://www.silverstonetek.com/product.php?pid=814&area=en)) `